packages/commonjs/test/test.js

```diff
@@ -258,6 +259,78 @@ test('import CommonJS module with esm property should get default export ', asyn
   t.is(result2.error.message, 'lib is not a function');
 });
 
+test('import CommonJS module with re-exported esm default should get default export', async (t) => {
```

Comment:
Could you do me a favour and instead of adding to this huge test file (which I would really hope to get rid of eventually, but we need it for some tricky caching things) and instead convert them to "function" tests? The reason is that for those tests, everything is neatly together in a separate folder and you do not have to scroll and search around for your fixtures.
Basically for each test, you add a folder to `test/fixtures/function` with your test fixtures (entry should always be `main.js`) and a `_config.js` file as well with something like

```js
module.exports: {
  description: '...some text to describe the test case',
  // options for the commonjs plugin
  pluginOptions: {
    defaultIsModuleExports: 'auto'
  }
}
```

Reply:
Of course, will do in the next day or two! Thanks for the speedy review :)